File: java/src/tarehart/rlbot/intercept/AerialChecklist.java
package tarehart.rlbot.intercept;

public class AerialChecklist extends LaunchChecklist {
    public boolean notSkidding;
    public boolean hasBoost;

    public boolean readyToLaunch() {
        return hasBoost && notSkidding && super.readyToLaunch();
    }
}

File: src/main/java/tarehart/rlbot/intercept/AerialChecklist.kt
package tarehart.rlbot.intercept

class AerialChecklist : LaunchChecklist() {
    var notSkidding: Boolean = false
    var hasBoost: Boolean = false

    override fun readyToLaunch(): Boolean {
        return hasBoost && notSkidding && super.readyToLaunch()
    }
}
